frontend/src/pages/landingPage/landingPage.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { groupBackground } from "../../assets/images/images";
+import { IconDownArrow } from "../../assets/images/images";
+import { Button } from "../../components/Buttons/Buttons";
+
+import "./landingPage.scss";
```

Comment:
This is causing an import bug. I sent an announcement over slack about it, but you might have missed it. I will paste it below so you can refer back to it whenever you need to.

> 
> @Sagun Shrestha @Narek Tsaturyan I think I know how some of the styles are being overridden. The main reason is that index.scss is compiled last, so any styles in that takes precedence over everything else. So instead of doing this for importing:
> 
> Footer.js
> ```
> import React, { Fragment } from "react";
> import { Button } from "../Buttons/Buttons";
> import "./_Footer.scss"; <----- DO NOT DO THIS!
> ```
> Do this instead:
> 
> index.scss'
> ```
> @use "components/normalize";
> @use "components/Basics" as *;
> @use "components/Buttons/Buttons" as *;
> @use "path/to/Footer.scss"; <------- DO THIS!
> ```
> This is my fault as I have been doing things the first way, but with your current and future issues please do it the second way. I will separately move all imports over to index.scss and fix any bugs that come up and merge the fix after your issues are done.

Reply:
I've downloaded your branch and taken a look. Is this the only thing you did or did you also perform the other things I outlined above and fixed any resulting bugs?

If all else fails, I have made a PR here in your repo to compare the changes I have made to make the button responsive. Feel free to download the branch and let me know if you are still experiencing bugs. In that case, please show me gifs or pictures so I can see what might be going on.

Reply:
<img width="1157" alt="Screen Shot 2022-02-28 at 11 22 07 AM" src="https://user-images.githubusercontent.com/45651986/156063694-be5b0762-9528-469d-94c9-547e37db5dab.png">

I only changed the import method to "index.scss" that you mentioned. Everything else is the same.

Reply:
Okay, I think It worked now! Thank you